.github/ISSUE_TEMPLATE/config.yml

```
@@ -2,7 +2,10 @@ blank_issues_enabled: true
 contact_links:
   - name: Question
     url: https://invite.slack.golangbridge.org/
-    about: Ask and answer questions in the #vscode Gopher slack channel
+    about: Ask and answer questions in the \#vscode Gopher slack channel
```

On second thoughts, shouldnt this be "or", not "and"? One would rarely both ask and answer at the same time :)

```suggestion
    about: Ask or answer questions in the \#vscode Gopher slack channel
```